cli/lsp/analysis.rs

```diff
@@ -1126,8 +1126,10 @@ impl CodeActionCollection {
 /// Prepend the whitespace characters found at the start of line_content to content.
 fn prepend_whitespace(content: String, line_content: Option<String>) -> String {
   if let Some(line) = line_content {
-    let whitespaces =
-      line.chars().position(|c| !c.is_whitespace()).unwrap_or(0);
+    let whitespaces = line
```

Comment:
`whitespace_index`?

Reply:
oh, I mean `non_whitespace_index`

Reply:
I guess it's like number of whitespace chars, probably fine as it is?

Reply:
The name sounds like it's a vector or something plural.